### PR TITLE
Fixed running tests with ant

### PR DESCRIPTION
**Pull request by lucasbuschlinger**

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -184,7 +184,7 @@
     <property name="opendiabetes.output.dir"
               value="${module.opendiabetes.basedir}/out/production/OpenDiabetes"/>
     <property name="opendiabetes.testoutput.dir"
-              value="${module.opendiabetes.basedir}/out/test"/>
+              value="${module.opendiabetes.basedir}/out/test/OpenDiabetes"/>
```
Comment (Magnusgaertner) [inline build.xml]: Why changing the output directory here? shouldn't it be out/test/

File: test/de/opendiabetes/tests/plugin/importer/TestImporterUtil.java [hunk 2]
```diff
@@ -9,10 +9,10 @@
 
 @Ignore
 public class TestImporterUtil {
-    public static Importer getImporterFromPath(String path){
-        PluginManager manager = new DefaultPluginManager();
-        manager.loadPlugin(Paths.get(path));
-        manager.startPlugins();
+    public static Importer getImporter(String importer){
+        PluginManager manager = new DefaultPluginManager(Paths.get("export"));
```
Comment (Magnusgaertner) [inline test/de/opendiabetes/tests/plugin/importer/TestImporterUtil.java:14]: this loads all plugins in export by default, thus makes testing plugins one by one harder, as many plugins get loaded extending the same Importer interface. I would prefer having both methods in Util, but using the old one for testing

Comment (Magnusgaertner) [reply test/de/opendiabetes/tests/plugin/importer/TestImporterUtil.java:14]: @ocastx, review please :)

Comment (Magnusgaertner) [reply test/de/opendiabetes/tests/plugin/importer/TestImporterUtil.java:14]: this issue seems to be fixed with the build script modifications, added assertions to test if only one plugin is loaded with the new version in
https://github.com/lucasbuschlinger/BachelorPraktikum/pull/15/commits/44f107390e90ea60338b88130522d4a06bc6ef52

File: build.xml (hunk 1 above)
Comment (lucasbuschlinger) [reply build.xml]: Yes it should, seems I tried something and forgot to revert this.

File: test/de/opendiabetes/tests/plugin/importer/TestImporterUtil.java (hunk 2 above)
Comment (lucasbuschlinger) [reply test/de/opendiabetes/tests/plugin/importer/TestImporterUtil.java:14]: So what now?
Is it fine or do you  want to revert to how it was?

File: build.xml (hunk 1 above)
Comment (lucasbuschlinger) [reply build.xml]: Done by cabcb05

**Comment by Magnusgaertner** [comment]
https://github.com/lucasbuschlinger/BachelorPraktikum/pull/16 should be reviewed and merged first

**Comment by lucasbuschlinger** [comment]
Yes, but what is the point with this PR then, if you merged tool into us2? 
Could be closed

File: test/de/opendiabetes/tests/plugin/importer/TestImporterUtil.java (hunk 2 above)
Comment (Magnusgaertner) [reply test/de/opendiabetes/tests/plugin/importer/TestImporterUtil.java:14]: Its fine ;)

**Pull request merged**